frontend/src/components/Banner/banner.js

```diff
@@ -0,0 +1,12 @@
+import './banner.scss';
+
+function Banner() {
+    return (
+        <div className="banner">
+            {/* TODO T#: replace banner / img */}
```

Comment:
almost there! Let's have you create an actual task in the tracker and then you can add the actual T number in here and other places below